capsules/detector_person_vehicle_bike_openvino/capsule.py

```diff
@@ -0,0 +1,25 @@
+from vcap import (
+    BaseCapsule,
+    NodeDescription,
+    DeviceMapper,
+    common_detector_options
+)
+from .backend import Backend
+
+
+class Capsule(BaseCapsule):
+    name = "detector_person_vehicle_bike_openvino"
+    description = ("OpenVINO person, vehicle, and bike detector. Optimized "
+                   "for surveillance camera scenarios.")
+    version = 1
+    device_mapper = DeviceMapper.map_to_all_myriad()
```

Comment:
Can you change this to map_to_openvino_devices() which is the new API for intelligently mapping onto CPU + any devices available. Currently it's the one where I block HDDL unless the environment variable is set.

Reply:
Sure, will do!